apps/researcher/src/messages/nl/messages.json

```diff
@@ -8,8 +8,8 @@
     "fetchError": "Er is een probleem opgetreden bij het ophalen van de resultaten. Laad de pagina opnieuw om het opnieuw te proberen.",
     "filters": "Filters"
   },
-  "Persons": {
-    "title": "{totalPersons, plural, =0 {0 personen} =1 {1 persoon} other {# personen}}",
+  "Constituents": {
+    "title": "{totalConstituents, plural, =0 {0 Personen en Instellingen} =1 {1 Persoon of Instelling} other {# Personen en Instellingen}}",
```

Comment:
Minor:
1. `personen en instellingen`, lowercased?
2. Instead of `totalConstituents` use `totalNumberOfConstituents` or `constituentsTotalCount`, to make clear this is a numeric value?